docusaurus/docs/publish-a-plugin/publishing-best-practices.md

```diff
@@ -67,6 +67,28 @@ Ensure your screenshots are a suitable resolution and file type (e.g. png, jpeg,
 
 :::
 
+**[Sponsorship link](../reference/plugin-json#links)**
+
+`info.links`
```

Comment:
non-blocking as can be follow up if necessary - but in a way its odd to call out sponsorship without talking about the other standard links we added here... we could talk about all of them in one go.